diff --git a/datasets_hoi/vcoco.py b/datasets_hoi/vcoco.py
index f30f0be..d89a5b7 100644
--- a/datasets_hoi/vcoco.py
+++ b/datasets_hoi/vcoco.py
@@ -6,7 +6,7 @@ import numpy as np
 import torch
 import torch.utils.data
 
-import datasets.transforms as T
+from . import transforms as T
 import clip
 from datasets_hoi.vcoco_text_label import *
 
diff --git a/util/slconfig.py b/util/slconfig.py
index 96aca13..e77140c 100644
--- a/util/slconfig.py
+++ b/util/slconfig.py
@@ -57,11 +57,11 @@ class SLConfig(object):
         [0, 1]
         >>> cfg = Config.fromfile('tests/data/config/a.py')
         >>> cfg.filename
-        "/home/kchen/projects/mmcv/tests/data/config/a.py"
+        "/path/to/config/a.py"
         >>> cfg.item4
         'test'
         >>> cfg
-        "Config [path: /home/kchen/projects/mmcv/tests/data/config/a.py]: "
+        "Config [path: /path/to/config/a.py]: "
         "{'item1': [1, 2], 'item2': {'a': 0}, 'item3': True, 'item4': 'test'}"
     """
     @staticmethod
